manifests/public.yaml

```diff
@@ -134,9 +134,6 @@ spec:
     - mountPath: /etc/salt/master.d/50-master.conf
       name: salt-master-config-master-conf
       readOnly: True
-    - mountPath: /etc/salt/master.d/50-api.conf
-      name: salt-master-config-api-conf
-      readOnly: True
```

Comment:
i'll run another test with only this change

Reply:
![salt-api](https://user-images.githubusercontent.com/5364817/38858528-4120ebe6-422c-11e8-8dfb-9575b4324aa8.png)

Reply:
we could just move this block https://github.com/kubic-project/salt/blob/master/config/master.d/50-api.conf#L4 to `50-master.conf` and only keep the `log_level` in `50-api.conf`

Reply:
> we could just move this block https://github.com/kubic-project/salt/blob/master/config/master.d/50-api.conf#L4 to 50-master.conf and only keep the log_level in 50-api.conf

That kinda defeats the purpose of having a config file for the master, and different one for the api. If we have common options, they should either be duplicated, or in a file that's common to both IMO